package.json

```diff
@@ -13,24 +13,30 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "clean": "rm -rf lib es",
+    "lint": "standard --verbose | snazzy",
     "build:commonjs": "BABEL_ENV=commonjs babel --quiet src --out-dir lib",
     "build:es": "BABEL_ENV=es babel --quiet src --out-dir es",
     "build": "npm run clean && npm run build:es && npm run build:commonjs",
-    "prepublish": "npm run clean && npm run build"
+    "prepublish": "npm run lint && npm run clean && npm run build"
   },
   "dependencies": {
-    "glamor": "^2.20.20"
+    "glamor": "2.20.24"
```

Comment:
Yeah essentially you're right! But we want to have as less dependencies as possible here. And I am pretty sure all dependencies we have added to `react-view` follow semver.
If that's not the case we should change it to pinned! :)